test/test_indsnetwork.jl

```diff
@@ -1,6 +1,7 @@
 using Dictionaries
 using ITensors
 using ITensorNetworks
+using ITensorNetworks: dim
```

Comment:
Could you explain this addition?

Reply:
I was founding this test errored without it because it was complaining it was not defined. Similar to the need to import `parent` above. I think this might be a Julia / compile/ Revise issue or something though because you are right you shouldn't need to explicitly import something that is already imported by Base

Reply:
`dim` is defined in `NDTensors` but reexported by `ITensors`, so it should get brought into the namespace by `using ITensors`.